shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/UTxO.hs

```diff
@@ -349,14 +324,14 @@ scriptsNeeded u tx =
   where
     unTxOut (TxOut a _) = a
     withdrawals = unWdrl $ getField @"wdrls" $ _body tx
-    UTxO u'' = (txinsScript (getField @"inputs" $ _body tx) u) ◁ u
-    -- u'' = Map.restrictKeys v (txinsScript (txins $ _body tx) u)  TODO
+    -- UTxO u'' = (txinsScript (getField @"inputs" $ _body tx) u) ◁ u
```

Comment:
can remove now

Reply:
Done. I redid this in the SetAlgebra style. so it is consistent with other uses of (◁)